pkg/graph/schema/types/mto_solution.graphql

```diff
@@ -39,12 +39,9 @@ type MTOSolution {
 input MTOSolutionChanges @goModel(model: "map[string]interface{}") {
   name: String # If sourced from a Common Solution, this will resolve from the common solution table (you can't edit this for common solution!)
   facilitatedBy: MTOFacilitator
+  neededBy: Time
   status: MTOSolutionStatus
   riskIndicator: MTORiskIndicator
-  key: MTOCommonSolutionKey
-  type: MTOSolutionType # If sourced from a Common Solution, this will resolve from the common solution table (you can't edit this for common solution!)
-  pocName: String
```

Comment:
good call for poc here. I'm a bit surprised they aren't editable, I have a follow up question about this in figma [here](https://www.figma.com/design/BcI7pNwfkmwaDD2EGGTsYC/IT-Lead?node-id=4339-101269&t=7VhEwAnqVOpk0jMH-4). We can reintroduce later if needed.

Reply:
@OddTomBrooks , I just checked with Zoë, poc and poc email should be update-able. Can you add that back into the changes object please?